KsApi/MockService.swift

```diff
@@ -1892,3 +1913,9 @@ private extension Result {
     }
   }
 }
+
+extension GraphAPI.CompleteOnSessionCheckoutMutation.Data: Decodable {
+  public init(from _: Decoder) throws {
+    fatalError("The test code should not actually be decoding this object.")
```

Comment:
This is required for us to use `performWithResult`. I'm fairly sure the `Decodable` requirement is for Apollo, not for the `MockGraphQLClient`, which doesn't actually do any decoding.